### ISSUE TITLE
Multi-output parameter for keras

### ISSUE THREAD

**Issue by widforss** [NVE/avalanche_ml#14]
Keras loss functions seems to work really bad when we don't compile a model that knows that it it a multi-output problem. The machine creators should have a parameter that specifies how many dims each output should have.

**Comment by widforss** [comment]
https://www.pyimagesearch.com/2018/06/04/keras-multiple-outputs-and-multiple-losses/
